Reference/google/protobuf/any.pb.swift

```diff
@@ -125,6 +125,14 @@ struct Google_Protobuf_Any: SwiftProtobuf.Message, SwiftProtobuf._MessageImpleme
     1: .unique(proto: "type_url", json: "typeUrl"),
     2: .same(proto: "value"),
   ]
+  internal var _storage = AnyMessageStorage()
+
+  internal mutating func _uniqueStorage() -> AnyMessageStorage {
```

Comment:
If you added a `typealias _Storage = AnyMessageStorage`, could you use the standard boilerplate for `_uniqueStorage` here instead of customizing it with the new return type?